PublicationsRouter/v2/oai-pmh/XWALK.md

```diff
@@ -24,3 +24,30 @@ Note that given the simplicity of the OAI_DC format, only a limited subset of th
 | metadata.publication_date | oai_dc:date |
 | metadata.license_ref | oai_dc:rights |
 | metadata.embargo | oai_dc:rights |
+
+Example Metadata Document, values in square brackets refer to the field in PubRouter's metadata, values in brackets indicate the cardinality of the field. 
+```xml
+<metadata xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/">
+	<oai_dc:dc xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
+		<dc:language>[metadata.article.language](0..n)</dc:language>
+		<dc:title>[metadata.article.title](1)</dc:title>
+		<dc:creator>[metadata.authors](1..n)</dc:creator>
+		<dc:subject>[metadata.article.subject](0..n)</dc:subject>
+		<dc:description>[metadata.article.version](0..1)</dc:description>
+		<dc:description>[metadata.publication_status](0..1)</dc:description>
+		<dc:description>[metadata.history_date](0..n)</dc:description>
+		<dc:description>[metadata.funding](0..n)</dc:description>
+		<dc:description>[provider.agent](1)</dc:description>
+		<dc:publisher>[metadata.journal.publisher](0..1)</dc:publisher>
+		<dc:contributor>[metadata.contributor](0..n)</dc:contributor>
+		<dc:identifier>[metadata.article.identifier](0..n)</dc:identifier>
+		<dc:source>[metadata.journal.identifier](0..n)</dc:source>
+		<dc:source>[metadata.journal.title](0..1)</dc:source>
+		<dc:type>[metadata.article.type](0..1)</dc:type>
+		<dc:date>[metadata.publication_date](1)</dc:date>
+		<dc:rights>[metadata.license_ref](0..n)</dc:rights>
+		<dc:rights>[metadata.embargo](0..1)</dc:rights>
+	</oai_dc:dc>
```

Comment:
I'm confused in the above example by this terminology: `...(0..n)</dc...` or `...(0..n)</dc...` etc.

Do you mean if we had say, 3 subjects we would end up with this:
```
<dc:subject>[metadata.article.subject #1],[metadata.article.subject #2],[metadata.article.subject #3]</dc:subject>
```
rather than this:
```
<dc:subject>[metadata.article.subject #1]</dc:subject>
<dc:subject>[metadata.article.subject #2]</dc:subject>
<dc:subject>[metadata.article.subject #3]</dc:subject>
```

Reply:
Also, if we are sending identifiers, do they get constructed something like this:
`<dc:identifier>doi:12345678</dc:identifier>`
`<dc:identifier>issn:1234-5678</dc:identifier>`
If so the example should show that.

Also, perhaps there should be an extra *Notes* column in the crosswalk table as with other xwalk,md pages.